Install best practices

efajardo opened the pull request:
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallBestPractices

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

efajardo commented:
@matyasselmeci can you please review?

matyasselmeci commented on docs/common/install-best-practices.md:
```diff
@@ -0,0 +1,139 @@
+
+CRLs are not distributed via `yum`. Instead, we provide the `fetch-crl` tool that downloads CRLs to the CA directory.
+
+<span class="twiki-macro INCLUDE" section="FetchCRLConfigProxy">InstallCertAuth</span>
```
Replace with link to `../common/ca/#managing-certificate-revocation-lists`

matyasselmeci commented on docs/common/install-best-practices.md:
```diff
@@ -0,0 +1,139 @@
+
+If you need a bit-for-bit, exact replica of your system prior to the upgrade, you'll want to use a backup strategy and restore from backup.
+
+Comments
```
Drop everything from here to the end.

matyasselmeci commented on docs/common/install-best-practices.md:
```diff
@@ -0,0 +1,139 @@
+
+<span class="twiki-macro INCLUDE" section="FetchCRLConfigProxy">InstallCertAuth</span>
+
+Removing Large numbers of RPMs
```
Drop this entire section:
- We don't support RHEL 5 anymore
- We don't ship yum-remove-osg anymore
- The rest of it isn't very useful

efajardo commented:
@matyasselmeci thanks for reviewing. I had already fixed most of those screen problems but forgot to commit them.  I fixed it now. can you review again?

matyasselmeci commented on docs/common/install-best-practices.md:
```diff
@@ -0,0 +1,112 @@
+Certificate Revocation List (CRL) Installation/Update
+-----------------------------------------------------
+
+CRLs are not distributed via `yum`. Instead, we provide the `fetch-crl` tool that downloads CRLs to the CA directory.
```
Could you add the link I mentioned?

efajardo replied:
Ok I think I did, but not sure it was the way you wanted.

matyasselmeci commented:
OK, looks good!
